File: MiniDOMFinanziaria/Finanziaria/Collaboratori/Premi/CSetPremiCursor.cs
using System;

namespace minidom
{
    public partial class Finanziaria
    {
        [Serializable]
        public class CSetPremiCursor : Databases.DBObjectCursor
        {
            private DBCursorField<bool> m_AScaglioni = new DBCursorField<bool>("AScaglioni");
            private DBCursorField<TipoIntervalloSetPremi> m_TipoIntervallo = new DBCursorField<TipoIntervalloSetPremi>("");
            private DBCursorField<TipoCalcoloSetPremi> m_TipoCalcolo = new DBCursorField<TipoCalcoloSetPremi>("");

            public CSetPremiCursor()
            {
            }

            public DBCursorField<TipoIntervalloSetPremi> TipoIntervallo
            {
                get
                {
                    return m_TipoIntervallo;
                }
            }

            public DBCursorField<TipoCalcoloSetPremi> TipoCalcolo
            {
                get
                {
                    return m_TipoCalcolo;
                }
            }

            public DBCursorField<bool> AScaglioni
            {
                get
                {
                    return m_AScaglioni;
                }
            }

            public override string GetTableName()
            {
                return "tbl_TeamManagers_SetPremi";
            }

            protected override Databases.CDBConnection GetConnection()
            {
                return Database;
            }

            protected override Sistema.CModule GetModule()
            {
                return null;
            }

            public override object InstantiateNew(DBReader dbRis)
            {
                return new CSetPremi();
            }
        }
    }
}

File: MiniDOMFinanziaria/Finanziaria/Collaboratori/Premi/CSetPremiCursor.vb
Imports minidom
Imports minidom.Databases
Imports minidom.Sistema
Imports minidom.Anagrafica



Partial Public Class Finanziaria


    <Serializable>
    Public Class CSetPremiCursor
        Inherits DBObjectCursor

        Private m_AScaglioni As New CCursorField(Of Boolean)("AScaglioni")
        Private m_TipoIntervallo As New CCursorField(Of TipoIntervalloSetPremi)("")
        Private m_TipoCalcolo As New CCursorField(Of TipoCalcoloSetPremi)("")

        Public Sub New()
        End Sub


        Public ReadOnly Property TipoIntervallo As CCursorField(Of TipoIntervalloSetPremi)
            Get
                Return Me.m_TipoIntervallo
            End Get
        End Property

        Public ReadOnly Property TipoCalcolo As CCursorField(Of TipoCalcoloSetPremi)
            Get
                Return Me.m_TipoCalcolo
            End Get
        End Property

        Public ReadOnly Property AScaglioni As CCursorField(Of Boolean)
            Get
                Return Me.m_AScaglioni
            End Get
        End Property

        Public Overrides Function GetTableName() As String
            Return "tbl_TeamManagers_SetPremi"
        End Function

        Protected Overrides Function GetConnection() As CDBConnection
            Return Finanziaria.Database
        End Function

        Protected Overrides Function GetModule() As CModule
            Return Nothing
        End Function

        Public Overrides Function InstantiateNew(dbRis As DBReader) As Object
            Return New CSetPremi
        End Function

    End Class




End Class
